store/storetest/team_store.go

```diff
@@ -668,6 +669,100 @@ func testTeamMembers(t *testing.T, ss store.Store) {
 	}
 }
 
+func testTeamMembersWithPagination(t *testing.T, ss store.Store) {
+	teamId1 := model.NewId()
+	teamId2 := model.NewId()
+
+	m1 := &model.TeamMember{TeamId: teamId1, UserId: model.NewId()}
+	m2 := &model.TeamMember{TeamId: teamId1, UserId: model.NewId()}
+	m3 := &model.TeamMember{TeamId: teamId2, UserId: model.NewId()}
+
+	if r1 := <-ss.Team().SaveMember(m1, -1); r1.Err != nil {
```

Comment:
As stated above: Please use testify

```suggestion
	r1 := <-ss.Team().SaveMember(m1, -1)
	require.Nil(t, r1.Err)
```